readthedocsext/theme/templates/projects/import_base.html

```diff
@@ -81,12 +81,43 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
                   {% if wizard.steps.prev %}
                     <button class="ui button"
                             name="wizard_goto_step"
-                            value="{{ wizard.steps.prev }}">{% trans "Previous" %}</button>
+                            value="{{ wizard.steps.prev }}">
+                      {% trans "Previous" %}
+                    </button>
                   {% endif %}
 
                   {% if wizard.steps.next %}
                     <input class="ui button primary" type="submit" value="{% trans "Next" %}" />
-                  {% else %}
+                  {% endif %}
+
+                  {% if wizard.steps.current == "config" %}
+                    <div class="ui small modal" data-modal-id="i-need-help">
+                      <div class="header">Add configuration file to your project</div>
+                      <div class="content">
+                        <p>These are the steps you need to follow:</p>
+                        <ol>
+                          <li>
+                            Create a <code>.readthedocs.yaml</code> in your project.
+                          </li>
+                          <li>Copy and paste the content of the example.</li>
+                          <li>Open a pull request with these changes.</li>
+                          <li>Merge the pull request in your project.</li>
```

Comment:
We could also avoid being specific here about pull requests. There is nothing on our side that needs this to be a pull request and users might not understand this.

```suggestion
                          <li>Merge this change to your project's repository.</li>
```